paddle/fluid/framework/ir/mkldnn/cpu_quantize_pass.cc

```diff
@@ -343,6 +343,65 @@ void CPUQuantizePass::QuantizePriorBox(Graph* graph) const {
                   quantize_prior_box_count);
 }
 
+void CPUQuantizePass::QuantizeTranspose(Graph* graph) const {
+  GraphPatternDetector gpd;
+  auto pattern = gpd.mutable_pattern();
+  patterns::Transpose transpose_pattern{pattern, name_scope_};
+  transpose_pattern();
+
+  int quantize_transpose_count = 0;
+  auto handler = [&](const GraphPatternDetector::subgraph_t& subgraph,
+                     Graph* g) {
+    VLOG(4) << "Quantize transpose op";
+    GET_IR_NODE_FROM_SUBGRAPH(transpose_op, transpose_op, transpose_pattern);
+    auto* transpose_op_desc = transpose_op->Op();
+
+    if (!transpose_op_desc->HasAttr("use_quantizer")) {
+      return;
+    }
+    // skip if should not be quantized
+    if (!transpose_op_desc->HasAttr("use_quantizer") ||
+        !boost::get<bool>(transpose_op_desc->GetAttr("use_quantizer"))) {
+      return;
+    }
+    GET_IR_NODE_FROM_SUBGRAPH(prev_op, prev_op, transpose_pattern);
+    GET_IR_NODE_FROM_SUBGRAPH(next_op, next_op, transpose_pattern);
+
+    // skip if prev op is not quantized
+    // in future we should checked if next_op is quantized
+    // transpose INT8 schould be used only between INT8 operators
```

Comment:
Why should transpose int8 be used only between int8 operators? I believe that the current implementation is better as performing an int8 transposition should take less time when performed on 8-bit rather than 32-bit primitives.

Reply:
This is not as significant acceleration when adding additional quantization or dequantization. We're already working on reshape2 int8 to get an unbroken int8 string.